includes/managers/widgets.php

```diff
@@ -338,6 +338,35 @@ public function ajax_get_widget_types_controls_config( array $data ) {
 			];
 		}
 
+		$config = $this->translate_default_values_to_site_locale( $config );
+
+		return $config;
+	}
+
+	public function translate_default_values_to_site_locale( $config, $path = WP_LANG_DIR . '/plugins/' ) {
+		$locale = Utils::get_current_locale();
+
+		if ( ! Utils::change_language_of_textdomain( $locale, $path ) ) {
+			return $config;
+		}
+
+		foreach ( $this->get_widget_types() as $widget_key => $widget ) {
+			// There is a bug in Elementor-Pro that causes the function get_stack() to fail.
+			// The bug in Elementor-Pro is fixed in 3.7.0.
+			// This is a temporary fix until Elementor-Pro 3.7.0 will be released.
+			if ( 'progress-tracker' === $widget_key || 'facebook-embed' === $widget_key ) {
```

Comment:
We need a real fix. Because the same problem may happen for other 3rd party widgets.